main.tf

```diff
@@ -18,7 +18,7 @@ data "aws_vpc" "default" {
   id = "${var.vpc_id}"
 }
 
-# Get all subnets from the necessary vpc
+# Get all subnets from the VPC
 data "aws_subnet_ids" "all" {
```

Comment:
Yes, `aws_subnet_ids` is a Terraform data source.
`https://www.terraform.io/docs/providers/aws/d/subnet_ids.html`